### ISSUE TITLE
Add tests to check that the components are visible 🧪 🌵 🥇 

### ISSUE THREAD

**Issue by oas004** [oas004/Material3Playground#27]
Add JVM UI tests for all the components on each page to check that they are present.

**Comment by oas004** [comment]
> Hey @oas004 just a quick question. I see you have used `assertExists` to check for visibility. Why not use `assertIsDisplayed`? 🤔

Hey! I think `assertIsDisplayed` would be the better choice on most of these tests since we are testing that they are actually displayed as you say. Feel free to use that 😊 I can't quite remember why we used `assertExists` to be honest.

Do you remember @fgiris? 😊

**Comment by fgiris** [comment]
It might be that if the composable is not in the tree then `assertExists` is the one that is used to see (e.g if you have if/else case for a composable) otherwise `assertIsDisplayed` should just be fine.

**Comment by oas004** [comment]
Yeah that could possibly be a reason @fgiris :) I think how it was done in #70 worked really nice though! :pray:
